### DISCUSSION TITLE
New To CloudLog. Testing It. Getting PHP Error

### DISCUSSION THREAD

**Discussion by parthapratimpalit** [magicbug/Cloudlog#1391] [General]
Hello Team,

This is VU3LXP. Name : Partha

I was checking out Cloudlog. Installation as below

Where Installed : 
Raspberry Pi 3b+
Installed Docker Inside It. 
Created a LAMP container ( Ubuntu 20.04.3 LTS,  Apache/2.4.41, MYSQL 8.0.27-0ubuntu0.20.04.1 , PHP v7.4.27, )
Installed latest CloudLog via Gitpull. 

The Cloudlog is blank and i just entered a dummy QSO in it for testing. Now asap i click the LOGBOOK button or CloudLOG icon its throwing me an PHP error.  Any idea what it might be ?

If i click CLOUDLOG button below error is coming

![image](https://user-images.githubusercontent.com/99000666/152474219-472c0c09-7164-41c7-94ad-c7044c3eb98c.png)

**************************************************************************************************************************************
Now if i click LOGBOOK button below error is coming

![image](https://user-images.githubusercontent.com/99000666/152473966-9f376b2e-3576-451a-b604-f3c9881a53f4.png)

**************************************************************************************************************************************

Rest all tabs are working fine

![image](https://user-images.githubusercontent.com/99000666/152473610-691ee654-5228-4ec0-9571-3b99e1c10ee9.png)

**Comment by magicbug** [comment]
Did you create a "Station Location" before saving a QSO?
